cypress/page/shipping-step.page.ts

```diff
@@ -3,8 +3,8 @@ class ShippingStepPage {
   private proceedCheckoutBtn: string;
 
   constructor() {
-    this.checkTerms = '#cgv';
-    this.proceedCheckoutBtn = '#form button';
+    this.checkTerms = '.checker input';
+    this.proceedCheckoutBtn = '.cart_navigation .standard-checkout';
```

Comment:
here you can use an attribute of a tag to indicate we are looking for the Proceed to checkout button